### PR TITLE
add the project folder as starting point

### PR DESCRIPTION
**Pull request by franzholz**
There is no description from where the commands can be started.

### CODE REVIEW

**Review by linawolf — DISMISSED** [review]
https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2473 must be resolved before new terms are spread throughout the documentation

**Review by DavidBruchmann — APPROVED** [review]

File: Documentation/ApiOverview/CommandControllers/Index.rst [hunk 1]
```diff
@@ -17,6 +17,14 @@ These commands can also be run from the TYPO3
 :ref:`scheduler <symfony-console-commands-scheduler>` if this option is not
 disabled in the :file:`Configuration/Services.yaml`.
 
+The starting point for the commands differs depending on the kind of your TYPO3 installation.  
+
+ 1. In installations with composer TYPO3 the starting point is usually the folder where the
```
Comment (brotkrueml) [inline Documentation/ApiOverview/CommandControllers/Index.rst:22]: This sentence is hard to understand. Additionally, it is at least misunderstanding: "starting point the folder where the file composer.json" -> there are many composer.json's in a typical TYPO3 installation. And no one wants to add var and vendor in a folder of an extension.

File: Documentation/ApiOverview/CommandControllers/Index.rst [hunk 2]
```diff
@@ -17,6 +17,14 @@ These commands can also be run from the TYPO3
...
+file :file:`composer.json` is resided, in this folder the directories :file:`var` and
+:file:`vendor` are also created. CLI-commands start usually with `vendor/bin/`.
+ 2. In legacy installations (without composer) the starting poin is usually the web-root,
```
Comment (brotkrueml) [inline Documentation/ApiOverview/CommandControllers/Index.rst:25]: Also here is a typo ("poin -> point"). Additionally "web-root" -> "web root"

File: Documentation/ApiOverview/CommandControllers/Index.rst [hunk 3]
```diff
@@ -17,6 +17,14 @@ These commands can also be run from the TYPO3
...
+ so that CLI-commands start with `typo3/vendor/bin/`.
```
Comment (brotkrueml) [inline Documentation/ApiOverview/CommandControllers/Index.rst:26]: CLI-commands -> CLI commands.

**Comment by brotkrueml** [comment]
Will provide a follow-up for this, as is it partially misunderstanding and contains typos.

**Comment by DavidBruchmann** [comment]
great, thanks!

**Pull request merged**